packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -359,10 +359,14 @@ class JITSnapshotter {
     final Directory outputDir = fs.directory(outputPath);
     outputDir.createSync(recursive: true);
 
-    final String engineVmSnapshotData = artifacts.getArtifactPath(Artifact.vmSnapshotData);
-    final String engineIsolateSnapshotData = artifacts.getArtifactPath(Artifact.isolateSnapshotData);
-    final String isolateSnapshotData = fs.path.join(outputDir.path, 'isolate_snapshot_data');
-    final String isolateSnapshotInstructions = fs.path.join(outputDir.path, 'isolate_snapshot_instr');
+    final String engineVmSnapshotData =
+        artifacts.getArtifactPath(Artifact.vmSnapshotData, null, buildMode);
+    final String engineIsolateSnapshotData =
+        artifacts.getArtifactPath(Artifact.isolateSnapshotData, null, buildMode);
+    final String isolateSnapshotData =
+        fs.path.join(outputDir.path, 'isolate_snapshot_data');
+    final String isolateSnapshotInstructions =
+        fs.path.join(outputDir.path, 'isolate_snapshot_instr');
```

Comment:
Wrap back to single-line.

[https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-line-length-of-80-characters](Style guide ref)

Reply:
Done